### ISSUE TITLE
Dataset download

### ISSUE THREAD

**Issue by Akhilesh64** [bircatmcri/MCIndoor20000#2]
Is the door dataset or the whole dataset still available for download ? If yes, can you please provide the link.

**Comment by lakonecna** [comment]
Hi! Were you able to access it anywhere?

**Comment by vigneshrajap** [comment]
I don't see any download links either. Is it still open-access?

**Comment by aptafti** [comment]
Hello, there,

I left Marshfield Clinic in 2018. It seems they removed the link, and said:

[image: image.png]

**Comment by vigneshrajap** [comment]
Hey,

Yes, I saw that. Is that any other means, you can pass on the dataset? 

It would be really helpful. Thanks.

**Comment by aptafti** [comment]
As I said, I left there 3 years ago. Not sure about their current strategy.

You may want to contact them directly

**Comment by h3ct0r** [comment]
@vigneshrajap did you had any success accessing the dataset?

**Comment by vigneshrajap** [comment]
No, I could not get it. But there are other indoor datasets like this one https://groups.csail.mit.edu/vision/datasets/ADE20K/

**Comment by h3ct0r** [comment]
> No, I could not get it. But there are other indoor datasets like this one https://groups.csail.mit.edu/vision/datasets/ADE20K/

Thank you, I was just looking at this dataset right now! Regards
